easyaudit/utils.py

```diff
@@ -96,3 +96,12 @@ def get_m2m_field_name(model, instance):
     for x in instance._meta.many_to_many:
         if x.related_model == model:
             return x.name
+
+
+def get_model_queryset(model):
+    queryset_method_name = getattr(model, "EASY_AUDIT_QUERYSET_METHOD", None)
```

Comment:
let's make `get_easyaudit_queryset` the default string,

```suggestion
    queryset_method_name = getattr(model, "EASY_AUDIT_QUERYSET_METHOD", "get_easyaudit_queryset")
    queryset_method = getattr(model, queryset_method_name, None)
```